### PR TITLE
Request for an asset implemented, However right now you wont see the ent...

### PR DESCRIPTION
**Pull request by kartikgupta0909**
...ry being added to the databases because we dont have any valid organisations

### CODE REVIEW

File: app/views/requests/requests.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<h2>Make an asset request</h1>
+<?php
+if ($requested !== null) {
```
Comment (nisargjhaveri) [inline app/views/requests/requests.php:3]: Change variable name!

File: app/views/requests/requests.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        echo "Submission failed";
+    }
+    echo "<br><br>";
+}
+?>
+<form action="" method="POST">
+    <fieldset>
+        <legend>Common details about the person</legend>
+        <label for="organisation_id">Organisation Id: </label><input id="organisartion_id" name="details[organisation_id]" required /><br>
+        <label for="asset_id">Asset: </label><input id="asset_id" name="details[asset_id]" required /><br>
```
Comment (nisargjhaveri) [inline app/views/requests/requests.php:17]: It doesn't make sense if someone directly have to enter `organisartion_id` and `asset_id`.

We need to think of a better way for this.

**Comment by nisargjhaveri** [comment]
Also, please use better commit messages.

**Comment by gauravmittal1995** [comment]
Please remove swp files. Use "git status" before adding to check which all files are set for additions.

File: app/views/requests/requests.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        <label for="date">Date : </label><input id="date" type="date" name="details[date]" /><br>
+	<label for="priotiy">Asset: </label><input id="priority" name="details[priority]" required /><br>
```
Comment (gauravmittal1995) [inline app/views/requests/requests.php:19]: "Asset: " is repeated.

**Comment by nisargjhaveri** [comment]
Please check #21, and update accordingly.

**Comment by nisargjhaveri** [comment]
This one seems like outdated. Closing.

**Pull request closed without merging**